ie_serving/models/local_model.py

```diff
@@ -52,7 +54,9 @@ def _get_mapping_config(cls, version):
         return None
 
     @classmethod
-    def get_engine_for_version(cls, model_name, version_attributes):
+    def get_engine_process_for_version(cls, model_name, version_attributes):
         engine_spec = cls._get_engine_spec(model_name, version_attributes)
-        engine = IrEngine.build(**engine_spec)
-        return engine
+        engine_process = multiprocessing.Process(target=IrEngine.build,
```

Comment:
Can we use `_start_engine_process_for_version` here as well to make it more consistent? (this one suggests that only model building will happen in this process)
I can see you made `_start_engine_process_for_version` for remote storage only (as it contains downloaded files cleanup). Can we have single entry point i.e. pass `_start_engine_process_for_version` for every model subclass?